packages/core/src/components/slider/sliderTrackStop.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as React from "react";
+import { Intent } from "../../common";
+
+export interface ISliderTrackStopProps {
+    value: number;
+    trackIntentAfter?: Intent;
+    trackIntentBefore?: Intent;
```

Comment:
thoughts on simply `intentAfter` and `intentBefore`? i feel like "before/after" suggests that it affects the track.

Reply:
I like it!